pkg/libvmi/status/status.go

```diff
@@ -141,6 +141,15 @@ func WithNodeName(node string) Option {
 	}
 }
 
+// WithTopologyHints sets the TopologyHints with a given TSCFrequency
+func WithTopologyHints(tscFrequency int64) Option {
+	return func(vmiStatus *v1.VirtualMachineInstanceStatus) {
+		vmiStatus.TopologyHints = &v1.TopologyHints{
+			TSCFrequency: &tscFrequency,
```

Comment:
Can you make it override only TSCFrequency and call it `WithTSCFrequency` instead?